package.json

```diff
@@ -13,17 +13,16 @@
     "update": "lernaupdate"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.2.0",
-    "@typescript-eslint/parser": "^4.2.0",
+    "@typescript-eslint/eslint-plugin": "^5.6.0",
+    "@typescript-eslint/parser": "^5.6.0",
     "coveralls": "^3.0.2",
-    "eslint": "^7.9.0",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "husky": "^4.3.0",
+    "eslint": "^8.4.1",
+    "husky": ">=6",
```

Comment:
Are these `>=`s for husky and lint-staged "safe" to use here?

Reply:
This is husky's install script's doing. Do you want me to pin it?

Reply:
Updated! I didn't know it was using [mrm](https://mrm.js.org/) for configuration, which seems like an interesting project. I should probably dig into it.